tests/common/logout_test.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var App = require('../lib/app');
+var assert = require('assert');
+
+describe('Logout', function() {
+  var app;
+  var loginView;
+  var servicesView;
+  this.timeout(30000);
+
+  before(() => {
+    app = new App();
+    return app.init()
+      .then(defaultView => { loginView = defaultView; });
+  });
+
+  after(() => {
+    return app.stop();
+  });
+
+  it('should logout', () => {
+    return loginView.loginSuccess(12345678).then(function(servicesView) {
+      return  servicesView.logoutSuccess().then(function(loginView) {
+        return loginView.isLoginButton();
```

Comment:
As a consequence of [this comment](https://github.com/fxbox/app/pull/17/files#r56498382), this line shoudn't be needed anymore.